### DISCUSSION TITLE
Schaltbild ist nicht gleich

### DISCUSSION THREAD

**Discussion by ZeroQool007** [frickelzeugs/FingerprintDoorbell#31] [Q&A]
Hallo, wollte gerade das Projekt nachbauen und beim Löten bzw davor ist mir aufgefallen, dass dein Schaltbild nicht mit einem WEMOS D1 Mini übereinstimmt. Könntest du mir bitte helfen. Danke

Oder funktioniert es generell nicht mit einem ESP8266?

![2022-11-13 12 07 08](https://user-images.githubusercontent.com/12768323/201518581-1028568b-87b8-4684-81c7-c5c5aebfd2c8.jpg)

**Comment by frickelzeugs** [comment]
Ja, das liegt daran, dass das ein ESP8266 ist, das Schaltbild ist aber für einen ESP32 (in der Mini Bauform). Offiziell läuft es aktuell nur auf einem ESP32, es gibt aber einen PR für den ESP8266, der aber noch nicht gemerged ist. Ich würde immer einen ESP32 empfehlen, da deutlich leistungsfähiger und die 2€ mehr spielen bei privaten Bastelprojekten keine Rolle. Es gibt seit kurzem auch ein Ticket zur Unterstützung von Flash encryption, dass ich sehr interessant finde. Da der esp8286 dies nicht unterstützt, würde er auch rausfallen.
